packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,55 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) to a
+directory in your runtime environment with the `filesystems` property. To access Amaozn EFS
+from lambda function, the Amazon EFS access point will be required. 
+
+Amazon EFS _access points_ are application-specific entry points into an EFS file system that
+make it easier to manage application access to shared datasets. Access points can enforce a user
+identity, including the user's POSIX groups, for all file system requests that are made through
+the access point. Access points can also enforce a different root directory for the file system
+so that clients can only access data in the specified directory or its subdirectories. See
+[Working with Amazon EFS Access Points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) for more details.
+
+The following sample enables the lambda function to mount the Amazon EFS filesystem via the `accessPoint` to `/mnt/msg` in the runtime environment. We have to create an access point in Amaozn EFS and make sure Lambda function has
+read write access to it.
+
+```ts
+// create a new Amaozn EFS filesystem
+const fileSystem = new efs.FileSystem(stack, 'Efs', { vpc });
+
+// create a new access point from the filesystem
+const accessPoint = fileSystem.addAccessPoint('AccessPoint', {
+  // set /mnt/msg as the root of the access point
+  path: '/mnt/msg',
+  // as /mnt/msg does not exist, the efs will create the directory with the following createAcl
+  createAcl: {
+    ownerUid: '1001',
+    ownerGid: '1001',
+    permissions: '750',
+  },
+  // enforce the POSIX identity so lambda function will access with this identity
+  posixUser: {
+    uid: '1001',
+    gid: '1001',
+  },
+});
+
+const fn = new lambda.Function(stack, 'MyLambda', {
+  code,
+  handler,
+  runtime,
+  vpc,
+  filesystems: [
+    lambda.FileSystem.fromEfsAccessPoint(accessPoint, '/mnt/msg'),
```

Comment:
This is not the same /mnt/msg as the one defined for the access point. Use different paths to make this clearer. Otherwise users might think that these are related and they are not.

Reply:
YES! You are right. Let me revise it again.

Reply:
Now I am exposing the access point with `/export/lambda` as the root and Lambda function to mount to  its `/mnt/msg`. Let's see if this looks better now.

Reply:
Yeah looks good.